locales/de.json

```diff
@@ -3105,7 +3103,7 @@
 			},
 			"headline": {
 				"createTopic": "Thema anlegen",
-				"privateTasks": "Private Aufgaben",
+				"draftTasks": "Entwürfe",
 				"savedTeachingMaterials": "Gespeicherte Unterrichtsmaterialien"
 			},
 			"input": {
```

Comment:
A few lines down, in line 3116/3114 there's an `"expandPrivateTasks": "Private Aufgaben ein\/ausklappen"` left